### PR TITLE
run_keybase: delete old flags from kbfsfuse and use the startup log file

### PR DESCRIPTION
**Pull request by oconnor663**
@gabriel

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **user_group-Chat_enabled** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1465934095099-f5be895f961c-2deddb94612e/user_group-Chat_enabled-f5be895f961c.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1465934095099-f5be895f961c-2deddb94612e/user_group-Chat_enabled-2deddb94612e.png)  
   user_group-Chat_enabled comparison

**Comment by oconnor663** [comment]
I sincerely doubt that, visdiff :)

@chromakode, known problem?

File: packaging/linux/run_keybase [hunk 1]
```diff
@@ -42,12 +42,13 @@ logdir="${XDG_CACHE_HOME:-$HOME/.cache}/keybase"
...
-keybase service &>> "$logdir/keybase.service.log" &
+keybase -d --log-file="$logdir/keybase.service.log" service &>> "$logdir/keybase.start.log" &
 echo Mounting /keybase...
-kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
-  -bserver bserver.kbfs.keybase.io:443 -log-to-file /keybase &>> "$logdir/keybase.startkbfs.log" &
+kbfsfuse -debug -log-to-file /keybase &>> "$logdir/keybase.start.log" &
 echo Launching Keybase GUI...
 /opt/keybase/Keybase &>> "$logdir/Keybase.app.log" &
+echo Launching updater...
+kbupdater -log-to-file &>> "$logdir/keybase.start.log" &
```
Comment (gabriel) [inline packaging/linux/run_keybase]: Don't need updater?

**Comment by gabriel** [comment]
lg2m if remove updater?

**Comment by gabriel** [comment]
We need this too... put in PR: https://github.com/keybase/client/pull/3098

Fixes the kbfs bin name in linux, which will make the watchdog work

File: packaging/linux/run_keybase (hunk 1 above)
Comment (oconnor663) [inline packaging/linux/run_keybase]: OOPS

**Pull request merged**